TheMET/TheMET/CatalogContentView/CatalogContentView.swift

```diff
@@ -74,11 +74,11 @@ class CatalogContentView: UIView, UICollectionViewDelegate, UICollectionViewData
                         if let image = image {
                             cell.backgroundState = .loaded(image)
                         } else {
-                            cell.backgroundState = .failed
+                            cell.backgroundState = .failed(.noInternet)
                         }
                     }
                 } else {
-                    cell.backgroundState = .failed
+                    cell.backgroundState = .failed(.noInternet)
```

Comment:
This state loading is failed not because there is no internet.  There is no image URL to load an image.